### ISSUE TITLE
Characters.getByGameName doesnt work on latest version

### ISSUE THREAD

**Issue by L3mmAa** [BrandonCookeDev/smashgg.js#29]
I tried to fetch the characters from 'super-smash-bros-ultimate'
But i constantly get error messages. It doesnt work with version '^3.0.2' but works with '2.3.8'

The code looks like this:
```
var smashgg = require('smashgg.js');
var { Character} = smashgg;

let meleeCharacters = Character.getByGameName('super-smash-bros-ultimate').then((e) => {
    console.log(e);
});
```

Error:
> (node:12524) UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'getByGameName' of undefined ...
(node:12524) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 1)(node:12524) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.

**Comment by BrandonCookeDev** [comment]
Ah, I think the Character class got left out in the TypeScript rewrite.
Will patch and get back to you soon.

**Comment by BrandonCookeDev** [comment]
Just Published 3.1.0 which adds the Character class back in. 
Apologies about the mixup, and thanks for logging an issue!
